README.md

```diff
@@ -1,38 +1,67 @@
 # NixOps
 
-NixOps (formerly known as Charon) is a tool for deploying NixOS
-machines in a network or cloud.
+[![Test](https://github.com/NixOS/nixops/workflows/CI/badge.svg)](https://github.com/NixOS/nixops/actions)
 
-* [Manual](https://hydra.nixos.org/job/nixops/master/tarball/latest/download-by-type/doc/manual)
-* [Installation](https://nixos.org/nixops/manual/#chap-installation) / [Hacking](https://hydra.nixos.org/job/nixops/master/tarball/latest/download-by-type/doc/manual#chap-hacking)
-* [Continuous build](http://hydra.nixos.org/jobset/nixops/master#tabs-jobs)
-* [Issue Tracker](https://github.com/NixOS/nixops/issues)
-* [Mailing list / Google group](https://groups.google.com/forum/#!forum/nixops-users)
-* [IRC - #nixos on freenode.net](irc://irc.freenode.net/#nixos)
+_NixOps_ is a tool for deploying [NixOS](https://nixos.org) machines in a network or cloud. Key features include:
 
-## Developing
+- **Declarative**: NixOps determines and carries out actions necessary to realise a deployment configuration.
+- **Testable**: Try your deployments on [VirtualBox](https://github.com/nix-community/nixops-vbox) or [libvirtd](https://github.com/nix-community/nixops-libvirtd).
+- **Multi Cloud Support**: Currently supports deployments to [AWS](https://github.com/NixOS/nixops-aws), [Hetzner](https://github.com/NixOS/nixops-hetzner) and [GCE](https://github.com/AmineChikhaoui/nixops-gce)
+- **Separation of Concerns**: Deployment descriptions are divided into _logical_ and _physical_ aspects. This makes it easy to separate parts that say _what_ a machine should do from _where_ they should do it.
+- **Extensible**: _NixOps_ is extensible through a plugin infrastructure which can be used to provide additional backends.
 
-To start developing on nixops, you can run:
+For more information please refer to the [NixOps manual](https://nixos.org/nixos/manual/).
 
-```bash
-  $ ./dev-shell
+### Installing
+
+_NixOps_ is included in nixpkgs and can be installed from your respective channel:
```

Comment:
We might should warn that this repo contains NixOps 2.0 and the channels don't package it yet?

Reply:
Oh, yes indeed I should add something to that effect indeed.

Reply:
So I added a Note below, let me know if that's OK for you.